packages/flutter_tools/lib/src/runner/flutter_command_runner.dart

```diff
@@ -273,12 +273,14 @@ class FlutterCommandRunner extends CommandRunner {
     // Check if the cwd is a flutter dir.
     while (directory.isNotEmpty) {
       if (_isDirectoryFlutterRepo(directory)) {
-        if (directory != Cache.flutterRoot) {
+        if (!_compareResolvedPaths(directory, Cache.flutterRoot)) {
           // Warn, but continue execution.
           printError(
-            'Warning - the active Flutter is not the one from the current directory:\n'
```

Comment:
Maybe: `the 'flutter' tool you are currently running is...`

To distinguish "Flutter" the project from "flutter" the tool.